sumologic/resource_sumologic_monitors_library_folder.go

```diff
@@ -141,6 +159,16 @@ func resourceSumologicMonitorsLibraryFolderRead(d *schema.ResourceData, meta int
 		return nil
 	}
 
+	fgpResponse, fgpErr := c.GetCmfFgp(fgpTargetType, folder.ID)
+	if fgpErr != nil {
+		// if FGP endpoint is not enabled (not implemented), we should suppress this error
+		if !HasErrorCode(fgpErr.Error(), "not_implemented_yet") {
```

Comment:
We should log a warning message here?

Reply:
Sure. We could log a warning here. 

As demonstrated in the other PR, the `setCmfFgp` operation would fail with an "not_implemented_yet" error, if a Terraform file has references to FGP.